test/electron/features/native-crash.feature

```diff
@@ -2,26 +2,29 @@ Feature: Native Errors
 
   Scenario: A minidump is uploaded on native error
     When I launch an app
+    And I wait for 1 session upload
```

Comment:
Do we need a new step for this? Could we do something like:

```suggestion
    Then the total requests received by the server matches:
      | sessions  | 1 |
```

Reply:
I _think_ I get it - since the session may take a second or two to send (depending on when the electron framework comes up), the hard crash means that the session may or may not be sent, **and** may or may not be persisted. So this step is trying to remove that variability?

Reply:
There's a 3 second wait in the "the total requests received by the server matches" step already. AFAICT the only difference with the new step is it waits an extra 1.9 seconds, but otherwise this would be the same

Reply:
The new step returns immediately upon delivery (rather than having a fixed delay), so it tends to be slightly quicker (not that this is a big deal).

It was also intended as a "softer" action rather than a strict assertion (although the result is basically the same).

Happy to change it around though.

Reply:
I don't think it's a big deal either way; it's a bit clearer with the new step so lets leave it like that